proxy/envoy/config.go

```diff
@@ -607,13 +612,40 @@ func buildClusters(services []*model.Service) []Cluster {
 				Ports:    []*model.Port{port},
 				Tags:     svc.Tags,
 			}
+
 			cluster := Cluster{
 				Name:             OutboundClusterPrefix + clusterSvc.String(),
 				ServiceName:      clusterSvc.String(),
 				Type:             "sds",
 				LbType:           DefaultLbType,
 				ConnectTimeoutMs: DefaultTimeoutMs,
 			}
+
+			if up, _ := upstreamsByHostname[svc.Hostname]; up != nil {
+				cluster.LbType = loadBalanceType(up.GetLbPolicy())
+				if cb := up.GetCircuitBreaker(); cb != nil {
+					if scb := cb.GetSimpleCb(); scb != nil {
+						cluster.OutlierDetection = &OutlierDetection{
+							ConsecutiveError:   int(scb.GetFailureThreshold()),
+							BaseEjectionTimeMS: int(1000 * scb.GetResetTimeoutSeconds()),
+						}
```

Comment:
needs more detailed treatment. See https://github.com/amalgam8/amalgam8/blob/master/sidecar/proxy/envoy/manager.go#L345 (345-370) for reference